curriculum/challenges/english/10-coding-interview-prep/rosetta-code/compare-a-list-of-strings.md

```diff
@@ -8,7 +8,7 @@ dashedName: compare-a-list-of-strings
 
 # --description--
 
-A list or sequence is an abstract data type that represents a finite number of ordered values, where the same value may occur more than once. Here is an example:
+A list is a unordered set of values that may contain duplicates. Here is an example:
```

Comment:
Ordered or unordered? If we are using an array here, then we have an ordering

Reply:
I should have done more research about [order theory](https://en.wikipedia.org/wiki/List_of_order_theory_topics), before I added your suggestion. 

I think, the nested array is [partially ordered](https://en.wikipedia.org/wiki/Partially_ordered_set), as far as I know they are the same but people without mathematical background, who think order is sort, will not be confused by this, and can keep their focus on the beating the challenge. 😓🤦